aconfigure.ac

```diff
@@ -823,6 +825,18 @@ else
 	fi
 	ac_android_cflags="$ac_android_cflags -DPJMEDIA_VIDEO_DEV_HAS_ANDROID=1"
 	;;
+  *mingw*)
+	if test "$enable_video" = "yes"; then
+	  ac_pjmedia_video=windows_os
+	  AC_SUBST(ac_pjmedia_video_dev_has_dshow)
+	  ac_pjmedia_video_dev_has_dshow=yes
+	  AC_SUBST(ac_v4l2_cflags)
+	  ac_v4l2_cflags="-DPJMEDIA_HAS_VIDEO=1 -DPJMEDIA_VIDEO_DEV_HAS_DSHOW=1"
+	  AC_SUBST(ac_v4l2_ldflags)
+	  ac_v4l2_ldflags=" -lstdc++ -lquartz -lole32 -loleaut32 -lrpcrt4 -lwinmm -luuid -lmingwex -lstrmiids "
```

Comment:
I'd like to draw a bit of attention that I'm somewhat abusing the v4l2 variables here, for the lack of e.g. ac_dshow_cflags. This might look unclean and/or confusing, but I think it is basically a small consmetic issue, if one thinks of "ac_v4l2_cflags" as like just "ac_camera_cflags". It does exactly that job, and obviously directshow camera and v4l2 camera can not possibly get in conflict in any way, so supposedly no breakage would happen.

Reply:
It would be better to create a separate variable ac_dshow_flags & ldflags. Just search the entire pjsip project for the string `ac_v4l2_cflags` to find out all the places it's being used and add the new variable.

Reply:
Ok, done.